PRIVACY_POLICY.md

```diff
@@ -0,0 +1,65 @@
+# Swarm-Extension Privacy Policy
+
+The Swarm Extension belongs to the Swarm Foundation with the purpose of offering web3 environment
+for running decentralised applications (dApps). It means, it provides similar functionalities as web2 browsers,
```

Comment:
This whole sentence is unclear to me. It extends the functionality of web2 browsers, so that sensitive information can be handled by the extension,. An example of such sensitive information being Bee client configuration or cross-domain localStore?